manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
+PackageVersion: 2022.5.15
+PackageLocale: en-US
+Publisher: Andrew Huynh
```

```suggestion
Publisher: athlabs
```
This must match Control Panel.